docs/future/vite.md

```diff
@@ -6,6 +6,8 @@ title: Vite
 
 [Vite][vite] is a powerful, performant and extensible development environment for JavaScript projects. In order to improve and extend Remix's bundling capabilities, we now support Vite as an alternative compiler. In the future, Vite will become the default compiler for Remix.
 
+The Vite plugin was stabilized as of Remix 2.7.0.
```

Comment:
Release notes already specify when Remix Vite was stabilized, but we're considering adding the version for when each feature is added. Not something we want to act on just yet.